Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -1291,7 +1343,7 @@ u64 SimulateDiscReadTime(u64 offset, u32 length)
 		}
 	}
 
-	g_last_read_offset = (offset + length - 2048) & ~2047;
+	g_last_read_offset = end_offset;
```

Comment:
No matter where we actually perform the decryption, we should still be able to account for the relevant timing differences: the IPCHLE code still knows that decryption is happening even if it doesn't happen there.